src/components/Inquiry/InquiryCompanySize.jsx

```diff
@@ -7,7 +7,7 @@ const InquiryCompanySize = () => {
   const { t } = useTranslation()
 
   return (
-    <div data-cy='company-size-container'>
+    <div  data-cy='company-size-container'>
```

Comment:
nitpicking but this file wouldn’t be here unless that dubble space right?

Reply:
addressed